ibm/mas_devops/roles/cp4d/README.md

```diff
@@ -1,13 +1,34 @@
 cp4d
 ====
 
-This role installs [IBM Cloud Pak for Data](https://www.ibm.com/uk-en/products/cloud-pak-for-data) Operator in the target cluster.
+This role installs or upgrades [IBM Cloud Pak for Data](https://www.ibm.com/uk-en/products/cloud-pak-for-data) Operator in the target cluster.
 
-The [cp4d_hack_worker_nodes](cp4d_hack_worker_nodes.md) role must have been executed during cluster set up to update the cluster's global image pull secret and reload all worker nodes.  Unfortunately Cloud Pak for data does not support using image pull secrets attached to service accounts in the namespace.
+Currently supported Cloud Pak for Data release versions supported are:
```

Comment:
minor: eliminate one of the "supported"  words

Reply:
fixed!